types/meteor/mongo.d.ts

```diff
@@ -128,44 +128,44 @@ declare module "meteor/mongo" {
             [id: string]: Number;
         }
 
+        type TransformFn = Function | null;
+
+        type Options = {
+            sort?: SortSpecifier;
+            skip?: number;
+            limit?: number;
+            fields?: FieldSpecifier;
+            reactive?: boolean;
+            transform?: TransformFn;
+        }
+
+        type DispatchTransform<Transf extends { transform?: TransformFn }, DefType, U> = Transf['transform'] extends null ? DefType : Transf['transform'] extends (...args: any) => any ? ReturnType<Transf['transform']> : U;
```

Comment:
How about using `TransformFn` as the first argument instead of `{ transform?: TransformFn }`?